src/Redux/rockets/Api.js

```diff
@@ -0,0 +1,19 @@
+/*eslint-disable*/
+
+import { createAsyncThunk } from '@reduxjs/toolkit';
+import axios from 'axios';
+
+const RocketApi = 'https://api.spacexdata.com/v4/rockets';
+
+const FetchApi = createAsyncThunk('get/rockets', async () => {
+  const Response = await axios.get(RocketApi);
+  const data = Response.data;
+  return data.map((Rocket) => ({
+    RocketId: Rocket.id,
+    RocketName: Rocket.rocket_name,
+    description: Rocket.description,
+    RocketImages: Rocket.flickr_images[0],
```

Comment:
- [ ] The name of the Rocket is not displaying. I suggest you check the API link to verify the corresponding name attribute

Reply:
Thank you for suggesting me. I will correct it 💯